Lib/fontmake/font_project.py

```diff
@@ -25,19 +25,12 @@
 from cu2qu.rf import fonts_to_quadratic
 from fontTools import subset
 from glyphs2ufo.glyphslib import build_masters, build_instances
+from mutatorMath.ufo import build as build_designspace
+from robofab.world import OpenFont
```

Comment:
> Once the pens are merged in cu2qu, defcon should work.

Yeah but the FontCollection stuff that generates compatible fonts in parallel won't work with a basic pen, will it?